control/control.leanos.xml

```diff
@@ -71,12 +71,12 @@ textdomain="control"
         </services_proposal>
 
 	<!-- self-update URL -->
-	<self_update_url>https://updates.suse.com/SUSE/Updates/SLE-INSTALLER/15-SP2/$arch/update</self_update_url>
+	<self_update_url>https://updates.suse.com/SUSE/Updates/SLE-INSTALLER/$os_release_version/$arch/update</self_update_url>
```

Comment:
I think we should add a comment that `$os_release_version` is replaced by `VERSION` from the `/etc/os-release` file in the inst-sys....

And that the `$arch` is replaced by the machine architecture...